cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -385,7 +385,7 @@ func (mh *MessageHandle) ListMessageWriteLoop(info *model.HubInfo, stopServe cha
 
 		obj, exist, _ := nodeListStore.GetByKey(key.(string))
 		if !exist {
-			klog.Errorf("nodeListStore for node %s doesn't exist", info.NodeID)
+			klog.Errorf("message with key %s doesn't exist", key.(string))
```

Comment:
object with key %s doesn't exist?

Reply:
nodeListStore store message waiting to send to edge node.